### ISSUE TITLE
Serving hv.Graph returns bokeh DeserializationError

### ISSUE THREAD

**Issue by hyamanieu** [holoviz/holoviews#5107]
#### ALL software version info
Happens on both windows & ubuntu machine shown below
##### windows:
`bokeh info:`

> Python version      :  3.9.7 (default, Sep 16 2021, 16:59:28) [MSC v.1916 64 bit (AMD64)]
> IPython version     :  7.27.0
> Tornado version     :  6.1
> Bokeh version       :  2.4.1
> BokehJS static path :  C:\Anaconda3\envs\lecann\lib\site-packages\bokeh\server\static
> node.js version     :  (not installed)
> npm version         :  (not installed)

holoviews 1.14.6
panel 0.12.4

##### ubuntu (jupyterhub):
`bokeh info`

> Python version      :  3.8.11 (default, Aug  3 2021, 15:09:35)
> IPython version     :  7.27.0
> Tornado version     :  6.1
> Bokeh version       :  2.4.0
> BokehJS static path :  /opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/server/static
> node.js version     :  (not installed)
> npm version         :  (not installed)

holoviews 1.14.6
panel 0.12.4

#### Description of expected behavior and the observed behavior

serving a `hv.Graph` leads to a `DeserializationError `(same on Ubuntu & Windows):

> ERROR:bokeh.server.protocol_handler:error handling message
>  message: Message 'PATCH-DOC' content: {'events': [{'kind': 'ModelChanged', 'model': {'id': '1601'}, 'attr': 'edge_renderer', 'new': {'id': '1599'}}], 'references': []} 
>  error: DeserializationError("Instance(GlyphRenderer) failed to deserialize reference to {'id': '1599'}")

#### Complete, minimal, self-contained example code that reproduces the issue

Taking examples from http://holoviews.org/reference/elements/bokeh/Graph.html#elements-bokeh-gallery-graph

```
import panel as pn
import numpy as np
import holoviews as hv
from holoviews import opts
hv.extension('bokeh')
pn.extension()
# Declare abstract edges
N = 8
node_indices = np.arange(N)
source = np.zeros(N)
target = node_indices

simple_graph = hv.Graph(((source, target),))

flowpane = pn.pane.HoloViews(simple_graph)
s = flowpane.show()
```
After it loaded, error pops out.

NB: in many cases the bokeh/panel server runs without problem, but at some point it crashes which is problematic.

##### Here is the Javascript console output:

> [bokeh] Unhandled ERROR reply to 36662A66CB1642BC91F426BE2B0333C7: DeserializationError("Instance(GlyphRenderer) failed to deserialize reference to {'id': '1599'}") bokeh.min.js:590:1103
> [bokeh] Unhandled ERROR reply to A65DCC3CAF4E4D078729C1C15E04F2FF: DeserializationError("Instance(GlyphRenderer) failed to deserialize reference to {'id': '1593'}") bokeh.min.js:590:1103

##### Here is the traceback in the notebook/ipython/python console

```

> WARNING:tornado.access:404 GET /favicon.ico (10.242.3.13) 0.35ms
> ERROR:bokeh.server.protocol_handler:error handling message
>  message: Message 'PATCH-DOC' content: {'events': [{'kind': 'ModelChanged', 'model': {'id': '1601'}, 'attr': 'edge_renderer', 'new': {'id': '1599'}}], 'references': []} 
>  error: DeserializationError("Instance(GlyphRenderer) failed to deserialize reference to {'id': '1599'}")
> Traceback (most recent call last):
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/server/protocol_handler.py", line 97, in handle
>     work = await handler(message, connection)
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/server/session.py", line 93, in _needs_document_lock_wrapper
>     result = func(self, *args, **kwargs)
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/server/session.py", line 287, in _handle_patch
>     message.apply_to_document(self.document, self)
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/protocol/messages/patch_doc.py", line 115, in apply_to_document
>     invoke_with_curdoc(doc, lambda: doc.apply_json_patch(self.content, setter))
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/document/callbacks.py", line 408, in invoke_with_curdoc
>     return f()
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/protocol/messages/patch_doc.py", line 115, in <lambda>
>     invoke_with_curdoc(doc, lambda: doc.apply_json_patch(self.content, setter))
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/document/document.py", line 391, in apply_json_patch
>     DocumentPatchedEvent.handle_json(self, event_json, references, setter)
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/document/events.py", line 259, in handle_json
>     handler(doc, event_json, references, setter)
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/document/events.py", line 467, in _handle_json
>     patched_obj.set_from_json(attr, value, models=references, setter=setter)
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/core/has_props.py", line 384, in set_from_json
>     descriptor.set_from_json(self, json, models=models, setter=setter)
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/core/property/descriptors.py", line 376, in set_from_json
>     value = self.property.prepare_value(obj, self.name, self.property.from_json(json, models=models))
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/core/property/instance.py", line 112, in from_json
>     raise DeserializationError(f"{self} failed to deserialize reference to {json}")
> bokeh.core.property.bases.DeserializationError: Instance(GlyphRenderer) failed to deserialize reference to {'id': '1599'}
> ERROR:bokeh.server.protocol_handler:error handling message
>  message: Message 'PATCH-DOC' content: {'events': [{'kind': 'ModelChanged', 'model': {'id': '1601'}, 'attr': 'node_renderer', 'new': {'id': '1593'}}], 'references': []} 
>  error: DeserializationError("Instance(GlyphRenderer) failed to deserialize reference to {'id': '1593'}")
> Traceback (most recent call last):
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/server/protocol_handler.py", line 97, in handle
>     work = await handler(message, connection)
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/server/session.py", line 93, in _needs_document_lock_wrapper
>     result = func(self, *args, **kwargs)
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/server/session.py", line 287, in _handle_patch
>     message.apply_to_document(self.document, self)
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/protocol/messages/patch_doc.py", line 115, in apply_to_document
>     invoke_with_curdoc(doc, lambda: doc.apply_json_patch(self.content, setter))
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/document/callbacks.py", line 408, in invoke_with_curdoc
>     return f()
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/protocol/messages/patch_doc.py", line 115, in <lambda>
>     invoke_with_curdoc(doc, lambda: doc.apply_json_patch(self.content, setter))
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/document/document.py", line 391, in apply_json_patch
>     DocumentPatchedEvent.handle_json(self, event_json, references, setter)
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/document/events.py", line 259, in handle_json
>     handler(doc, event_json, references, setter)
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/document/events.py", line 467, in _handle_json
>     patched_obj.set_from_json(attr, value, models=references, setter=setter)
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/core/has_props.py", line 384, in set_from_json
>     descriptor.set_from_json(self, json, models=models, setter=setter)
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/core/property/descriptors.py", line 376, in set_from_json
>     value = self.property.prepare_value(obj, self.name, self.property.from_json(json, models=models))
>   File "/opt/tljh/user/envs/devenv/lib/python3.8/site-packages/bokeh/core/property/instance.py", line 112, in from_json
>     raise DeserializationError(f"{self} failed to deserialize reference to {json}")
> bokeh.core.property.bases.DeserializationError: Instance(GlyphRenderer) failed to deserialize reference to {'id': '1593'}

```

**Comment by hyamanieu** [comment]
Thanks for putting this up in the next milestone.

I amended my comment, I just saw I forgot to import panel although I am using it for simplicity (but the bug shows without it)

**Comment by philippjfr** [comment]
Looks like this was a bokeh issue, fixed in bokeh 2.4.2.
